### DISCUSSION TITLE
Print release notes in napari-hub-bot zulip

### DISCUSSION THREAD

**Discussion by haesleinhuepf** [chanzuckerberg/napari-hub#492] [Ideas]
Hi all,

when a napari-plugin has a new release, I typically see that in the [napari-hub-bot zulip chat](https://napari.zulipchat.com/#narrow/stream/290453-hub-updates). I'm just wondering if it might be possible to also post the release notes of the new release there? This would spare me some clicks and could motivate developers to actually maintain release notes.

Thanks!

Best,
Robert

**Comment by neuromusic** [comment]
@haesleinhuepf what would be the most reliable place to pull this info? from the tagged release on github, such as https://github.com/haesleinhuepf/napari-time-slicer/releases/tag/0.4.4 ?

Which would generalize to `https://github.com/{repository}/releases/tag/{version}`?

**Reply by haesleinhuepf** [reply]
Correct. That would be fantastic.

**Reply by haesleinhuepf** [reply]
One more hint: You can use the github API to read out release notes: https://docs.github.com/en/rest/releases/releases#get-a-release

**Reply by lgan-czi** [reply]
Hi @haesleinhuepf! Would something like this be good?
[napari-time-slicer](https://napari-hub.org/plugins/napari-time-slicer) leads to the hub, and clicking on [0.4.8](https://github.com/haesleinhuepf/napari-time-slicer/releases/tag/0.4.8) leads to the release notes
<img width="741" alt="Screen Shot 2022-07-05 at 4 20 09 PM" src="https://user-images.githubusercontent.com/106401390/177432497-0c4ce7d4-0f5b-4479-bb19-cc1d561e94b6.png">

**Reply by haesleinhuepf** [reply]
That looks absolutely fantastic @lgan-czi ! I love it 🤩

Maybe call it "release notes" like GitHub does and not "patch notes" (I'm not sure what a patch is)

**Reply by lgan-czi** [reply]
Thanks, will do!

**Reply by lgan-czi** [reply]
Hi @haesleinhuepf! Apologies for the delay, but I just merged the Zulip bot changes, new messages should start showing up in the next release of napari-hub.

**Comment by haesleinhuepf** [comment]
I've just seen this the very first time online and I love it. Thanks everyone involved! @lgan-czi @richaagarwal @neuromusic 

![image](https://user-images.githubusercontent.com/12660498/186164074-9a57e209-1903-4f3b-894e-d1f8bc733245.png)
